show_client/buffer_pool_cli.go

```diff
@@ -0,0 +1,110 @@
+package show_client
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	log "github.com/golang/glog"
+	sdc "github.com/sonic-net/sonic-gnmi/sonic_data_client"
+)
+
+// BufferPoolWatermarkResponse mirrors the CLI table: Pool -> Bytes
+// Example output: {"egress_lossless_pool":{"Bytes":"9216"}, ...}
+
+type BufferPoolStat struct {
+	Bytes string `json:"Bytes"`
+}
+
+// Constants / schema strings
+// Runtime schema: COUNTERS_BUFFER_POOL_NAME_MAP is a single hash mapping pool_name -> oid:0x...
+// Watermark tables: USER_WATERMARKS / PERSISTENT_WATERMARKS (keys: <table>:oid:<hex>)
+const (
+	userWatermarkTable       = "USER_WATERMARKS"
+	persistentWatermarkTable = "PERSISTENT_WATERMARKS"
+	bufferPoolNameMapKey     = "COUNTERS_BUFFER_POOL_NAME_MAP"
+	logPrefix                = "[buffer_pool] "
```

Comment:
nonblocking: Since this is a const that is visible at the package level, update this var to a buffer pool specific naming like "bufferPoolLogPrefix"

Reply:
Discussed with @Guoyuan001, he will take changes in next PR